api/src/controllers/place.js

```diff
@@ -85,7 +85,7 @@ router.get(
 
     const data = await Place.findAll({
       ...query,
-      attributes: ["_id", "encrypted", "encryptedEntityKey", "organisation", "createdAt", "updatedAt", "deletedAt"],
+      attributes: ["_id", "encrypted", "encryptedEntityKey", "createdAt", "updatedAt", "deletedAt"],
```

Comment:
```suggestion
      attributes: ["_id", "encrypted", "encryptedEntityKey", "organisation", "createdAt", "updatedAt", "deletedAt"],
```

pour l'instant ça nous est bien utile pour voir les problèmes de cache entre organisations (https://mano-20.sentry.io/issues/5646516318/?project=4506672157229056&query=%22DataLoader%3A+personsFromOtherOrgs%22&referrer=issue-stream&statsPeriod=14d&stream_index=0)

je suggèrerai de 
- conserver pour les personnes
- voir le problème de cache
- régler le problème de cache
- supprimer aussi pour les personnes

Reply:
Du coup c'est plus utile ça ? Je me demande si je vais pas réssussiter cette PR :)